hack/local-up-cluster.sh

```diff
@@ -738,6 +744,16 @@ clientConnection:
 hostnameOverride: ${HOSTNAME_OVERRIDE}
 featureGates: ${FEATURE_GATES}
 EOF
+    if [ "${KUBEPROXY_MODE}" == "ipvs" ]; then
```

Comment:
wouldn't it be better to just include this in the creation of the file above?

e.g.
```
mode:"${KUBEPROXY_MODE}
```

Reply:
Right now the default of this var will be “”. Would kubeproxy accept `mode: ` if this was unspecified?

Reply:
> Would kubeproxy accept mode: if this was unspecified?

Yes. Kube-proxy treat the empty mode as default mode - the iptables mode.

Reply:
I sent a PR to clean up this, ref: https://github.com/kubernetes/kubernetes/pull/54738